### DISCUSSION TITLE
Ban all script

### DISCUSSION THREAD

**Discussion by VeryWellEducatedBoy** [hxr404/Discord-Console-hacks#9] [Ideas]
not really a "hack" but i found this somewhere and wanted to share so if you want you can add this script (not mine) 
only works in browser since it uses `prompt()`

i didint know if i should post this suggestion here or in \#suggestions in the discord server, so i posted here

```js
/*
        BAN ALL SCRIPT 
        copy and paste this script into the console
        put the server id in the prompt and it will ban everyone possible in the server (if you have permissions obviously)
*/

var findModule = (item) => Object.values(webpackJsonp.push([[],{['']:(_,e,r)=>{e.cache=r.c}},[['']]]).cache).find(m=>m.exports&&m.exports.default&&m.exports.default[item]!==void 0).exports.default;var serverId = prompt('Server Id');var members = findModule('getMembers').getMembers(serverId);var { banUser } = findModule('banUser');var count = 0;
while (true) {if ((count % 40) === 0) await new Promise(_ => setTimeout(_, 2000));console.log(" --> Banned "+members[count].userId);banUser(serverId, members[count].userId);count++;}
```

**Comment by steinhorror** [comment]
Not working bc of Discord internal change from 22nd October

**Comment by RobinRMC** [comment]
This will 99.99% sure get your Discord account banned.
This thread should be locked, due to the lack of permission from the author to share it.

**Reply by ghost** [reply]
really? oh my GOD. its almost like 99.99% of the scripts in the repo will get your Discord account banned. thanks for trying to help though.

**Reply by RobinRMC** [reply]
Most of them won't get you banned. The script mentioned in this thread will most definitely get your account terminated.

**Reply by hxr404** [reply]
This is outdated anyway, so it doesn't really matter.
